src/protagonist/API/Infrastructure/Messaging/AssetModificationRecord.cs

```diff
@@ -13,22 +13,25 @@ public class AssetModificationRecord
     public Asset? Before { get; }
     public Asset? After { get; }
     
+    public bool AssetModifiedEngineNotified { get; }

```

Comment:
As this is a property of `AssetModificationRecord` is the `AssetModified` prefix redundant? `AssetModificationRecord.AssetModifiedEngineNotified` vs `AssetModificationRecord.EngineNotified`.

This new property is specific to `ChangeType.Update` notifications. `DeleteFrom` is specific to `ChangeType.Delete` notifications - should be be cleverer about how `ChangeType`-specific information is included, rather than adding a new property every time?